user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Customizing_the_layout_of_notification_messages.md

```diff
@@ -78,6 +78,87 @@ Example: The template called “text-notifications” should contain the templat
 
 The following example shows a *NotifyTemplates.xml* file in which two templates have been defined: one for plain-text notifications and another one for HTML notifications:
 
+### [From DataMiner 10.5.0 [CU0] onwards](#tab/tabid-1)
```

Comment:
Technically this will already be present from 10.5.12+, but I agree that 10.5+ looks cleaner

Reply:
We try to always add both the feature release and main release version, so that the documentation is clear regardless of which release track the user is on. I've adjusted this in the PR.